README.md

```diff
@@ -22,20 +22,22 @@ For VSCode, you should see a popup in the bottom right when you open the project
 
 ### Running the Dev Server
 
-To run the development server, you first need to have `npm`/`pnpm`/`yarn` installed in your system.
-Only `pnpm` is regularly used and tested for the site, so you might encounter bugs or incompatibilities using an alternate package manager. [Instructions on how to install `pnpm` can be found on its official installation page.](https://pnpm.io/installation)
+To run the development server, you first need to have `npm` installed in your system. Usually this comes pre-installed with [Node.js](https://nodejs.org), so you might want to install that first.
 
-Once `pnpm` is installed, clone this repository, navigate into the project folder and run `pnpm i` to install dependencies:
+> [!NOTE]
```

Comment:
cool use of Github's admonition thingy, could be a warning though maybe ?

Reply:
Dont think it is that important tbh

Reply:
true